### PR TITLE
Allow per:Object in ReplicationMixinNodeTypesOverride

### PR DESCRIPTION
**Pull request by reggie7**
Batch of changes for #658. The full fix will come in smaller chunks easier to consume.

Content is fully described in the title.

### CODE REVIEW

File: platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/com.peregrine.replication.impl.ReplicationMixinNodeTypesOverride.xml [hunk 1]
```diff
@@ -25,5 +25,5 @@
   -->
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
     jcr:primaryType="sling:OsgiConfig"
-    replicationNodeTypes="[per:AssetContent,per:PageContent]"
+    replicationNodeTypes="[per:AssetContent,per:PageContent,per:Object]"
```
Comment (reusr1) [inline platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/com.peregrine.replication.impl.ReplicationMixinNodeTypesOverride.xml:28]: are we missing a couple of other types here as well? (nt:file, nt:folder, sling:Folder, per:ObjectDefinition, others?)

**Comment by cmrockwell** [comment]
While I see the notice says success after publishing or unpublishing an object. 

Screen Shot 2021-02-19 at 3 56 01 PM

I also see two problems
1. Web Publishing Tab does not populate publishing information
2. JS errors prevent navigating into nested objects, such as clicking 'allfields' from http://localhost:8080/content/admin/pages/objects.html/path:/content/example/objects 

Is this something to fix here or leave for another PR?

**Comment by reggie7** [comment]
Thanks for seeing into this @cmrockwell, the change is so small that I am sure it's not causing any new issues, just exposing existing ones. We can simply wait with this one for the main PR that I will add next week.

**Review by flx-sta — APPROVED** [review]
@reggie7 please open an issue for the follow-up issues

**Comment by reggie7** [comment]
@Felix-Puetz #727 already describes this one:
> 1. Web Publishing Tab does not populate publishing information

@cmrockwell please create an issue for the other case, I don't get the `js` error:
> 2. JS errors prevent navigating into nested objects, such as clicking 'allfields' from http://localhost:8080/content/admin/pages/objects.html/path:/content/example/objects

**Review by reusr1 — APPROVED** [review]

File: platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/com.peregrine.replication.impl.ReplicationMixinNodeTypesOverride.xml (hunk 1 above)
Comment (reggie7) [reply platform/base/ui.apps/src/main/content/jcr_root/apps/runmodes/config/com.peregrine.replication.impl.ReplicationMixinNodeTypesOverride.xml:28]: Hard to say, depends more on our intentions I guess. I think we could have unwanted side-effects if we add those as well. It requires proper testing and more insight. We could first assess #773's results and view the above configuration from functional perspective.

**Pull request merged**